include/boost/geometry/srs/projections/proj/omerc.hpp

```diff
@@ -78,14 +79,16 @@ namespace projections
 
                 // FORWARD(e_forward)  ellipsoid
                 // Project coordinates from geographic (lon, lat) to cartesian (x, y)
-                inline void fwd(Parameters const& par, T const& lp_lon, T const& lp_lat, T& xy_x, T& xy_y) const
+                inline void fwd(Parameters const& par, T const& lp_lon, T const& lp_lat, T& xy_x,
+                                T& xy_y) const
                 {
                     static const T half_pi = detail::half_pi<T>();
 
                     T  s, t, U, V, W, temp, u, v;
 
                     if (fabs(fabs(lp_lat) - half_pi) > epsilon) {
-                        W = this->m_proj_parm.E / math::pow(pj_tsfn(lp_lat, sin(lp_lat), par.e), this->m_proj_parm.B);
+                        W = this->m_proj_parm.E / math::pow(pj_tsfn(lp_lat, sin(lp_lat), par.e),
+                            this->m_proj_parm.B);
```

Comment:
This alignment looks confusing because this is not the rest of the expression but a parameter of `pow`. I'd align it with `pj_tsfn` line above. What do you think?

Reply:
I agree.

Reply:
However if this was taken from the original proj then I'd leave it as it is.